### PR TITLE
Parse query steps in postgresql slowlog

### LINKED ISSUE #13496: [FileBeat][Postgresql] Parse <unnamed> query log format
**Describe the enhancement:**
As far as i can tell postgresql module doesn't support format with `parse <unnamed>`:
```
2019-09-04 15:52:38.004 CEST [31136] user@host LOG:  duration: 12.437 ms  parse <unnamed>: select * from table
```
It requires a statement keyword which is not always present so values are not parsed correctly

**Describe a specific use case for the enhancement or feature:**
Right now users are forced to create custom injests and it takes time...

### PR DESCRIPTION
**Pull request by jsoriano**
Slow logs in PostgreSQL can log information per query step, parse the
query step and the assigned query name if present.

Fix #13496

### CODE REVIEW

File: filebeat/module/postgresql/log/test/postgresql-query-steps-slowlog.log-expected.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        "event.duration": 12437000,
+        "event.module": "postgresql",
+        "event.timezone": "CEST",
+        "fileset.name": "log",
+        "input.type": "log",
+        "log.level": "LOG",
+        "log.offset": 0,
+        "message": "2019-09-04 15:52:38.004 CEST [31136] user@host LOG:  duration: 12.437 ms  parse <unnamed>: select * from table",
+        "postgresql.log.database": "host",
+        "postgresql.log.query": "select * from table",
```
Comment (kaiyan-sheng) [inline filebeat/module/postgresql/log/test/postgresql-query-steps-slowlog.log-expected.json]: Should this be documented? If `postgresql.log.query_name` is <unnamed>, then it won't show up in the json output.

Comment (kaiyan-sheng) [reply filebeat/module/postgresql/log/test/postgresql-query-steps-slowlog.log-expected.json]: Sorry, some reason the `<unnamed>` part didnt show up. Like under the documented `- name: query_name` description, do we need to add something like `Name given to a query when using extended query protocol. If it is <unnamed>, then this field will be ignored`.

Comment (jsoriano) [reply filebeat/module/postgresql/log/test/postgresql-query-steps-slowlog.log-expected.json]: Oh, you mean to document the behaviour when no name is present, ok, I'll do it!

Comment (jsoriano) [reply filebeat/module/postgresql/log/test/postgresql-query-steps-slowlog.log-expected.json]: Oh, I had extended the description but I prefer yours, changing it.

**Pull request merged**